utils/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
+        "target": "ES6" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
+        "module": "ES2022" /* Specify what module code is generated. */,
```

Comment:
Do we need to test both for common js?

Reply:
We already have simple compatibility tests, i thing running the same for each platform is a bit an overkill.